stdlib/2and3/bz2.pyi

```diff
@@ -20,7 +21,7 @@ if sys.version_info >= (3, 3):
              errors: Optional[str] = ...,
              newline: Optional[str] = ...) -> IO[Any]: ...
 
-class BZ2File(BinaryIO):
+class BZ2File(io.BufferedIOBase, IO[bytes]):  # type: ignore  # python/mypy#5027
```

Comment:
At runtime it inherits from `_compression.BaseStream`, which is a subclass of `BufferedIOBase` that adds some private methods.